@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -682,11 +682,11 @@ export interface MarkerTechniqueParams extends BaseTechniqueParams {
     /**
      * Text position regarding the baseline.
      */
-    hAlignment?: "Left" | "Center" | "Right";
+    hAlignment?: DynamicProperty<"Left" | "Center" | "Right">;
```

Comment:
How do you interpolate between strings?

Reply:
Only discrete interpolation is supported right now.

Reply:
Makes sense, DynamicProperty is a bit of a misleading name, it is a property that is based on some level correct?